shotover-proxy/src/transforms/cassandra/sink_cluster/token_map.rs

```diff
@@ -47,7 +55,7 @@ mod test_token_map {
         vec![
             CassandraNode::new(
                 "127.0.0.1:9042".parse().unwrap(),
-                "rack1".into(),
+                "dc1".into(),
```

Comment:
this arg is still for the rack right?

Reply:
Oh I don't know how I got that mixed up. Thanks